app/QueryGenerator.php

```diff
@@ -1003,7 +1003,9 @@ public function loadJoin()
 		}
 		if ($this->searchFieldsForDuplicates) {
 			$duplicateCheckClause = [];
+			$stateFromMainQuery = (null === $this->stateCondition) ? 'All' : $this->stateCondition;
 			$queryGenerator = new self($this->moduleName, $this->user->getId());
+			$queryGenerator->setStateCondition($stateFromMainQuery);
```

Comment:
The function takes a string as a function argument, and you can also pass an int here, which is an error